test/test_cli.py

```diff
@@ -389,11 +395,11 @@ def test_severity(self, capsys, caplog):
         # Check command line parameters - wrong case
         with pytest.raises(SystemExit) as e:
             main(["cve-bin-tool", "-S", "HIGH", self.tempdir])
-        assert e.value.args[0] == ERROR_CODES[SystemExit]
+        assert e.value.args[0] == 2
         # Check command line parameters - wrong option
         with pytest.raises(SystemExit) as e:
             main(["cve-bin-tool", "-S", "ALL", self.tempdir])
-        assert e.value.args[0] == ERROR_CODES[SystemExit]
+        assert e.value.args[0] == 2
```

Comment:
It's a fine line between 'make tests DRY and human-readable' and 'avoid depending on internals in tests'.

In this case I think this is an improvement, since `SystemExit` may carry various exit codes.